app/controllers/files_controller.rb

```diff
@@ -49,13 +49,20 @@ def create
   def update
     @repository_file = resource_class.create(params.merge(user: current_user))
     if resource.valid?
-      flash[:success] = "Successfully changed the file."
+      flash[:success] = "Successfully changed the file #{params[:path]}."
       redirect_to fancy_repository_path(repository, path: resource.target_path)
     else
       render :show
     end
   end
 
+  def destroy
```

Comment:
I'm not sure if this is a hound/rubocop bug or I just don't understand this comment. There is no branching in this method.